bundle/definition/validation_test.go

```diff
@@ -42,6 +42,39 @@ func TestObjectValidationValid(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestObjectValidationValid_CustomValidator(t *testing.T) {
+	s := `{
+		"type": "object",
+		"properties" : {
+			"file" : {
+				"type": "string",
+				"contentEncoding": "base64"
+			}
+		},
+		"required" : ["file"]
+	}`
+	definition := new(Schema)
+	err := json.Unmarshal([]byte(s), definition)
+	require.NoError(t, err, "should have been able to marshal definition")
+	assert.Equal(t, "object", definition.Type, "type should have been an object")
+	props := definition.Properties
+	assert.NotNil(t, props, "should have found properties")
+	assert.Equal(t, 1, len(props), "should have had a single property")
+	propSchema, ok := props["file"]
+	assert.True(t, ok, "should have found file property")
+	assert.Equal(t, "string", propSchema.Type, "file type should have been a string")
+	assert.Equal(t, "base64", propSchema.ContentEncoding, "file contentEncoding should have been base64")
+
+	val := struct {
+		File string `json:"file"`
+	}{
+		File: "SGVsbG8gV29ybGQhCg==",
```

Comment:
If I change this to an invalid base64 value such as "SGVsbG8gV29ybGQhCg===", the test still passes. Shouldn't it fail?

Reply:
Good call; indeed, as mentioned in the code comments, the current validator is a no-op (simply _allows_ for a `contentEncoding` property), but we can and perhaps should actually perform some basic validations specific to this property.  Preference on doing so in this PR or as a follow-up?

Reply:
Actually, I'd prefer it in this PR to check feasibility _before_ merging this PR. Hope it's not too much work - maybe a separate commit?

Reply:
I think the values we need to support are defined by https://json-schema.org/latest/json-schema-validation.html#rfc.section.8.3.

Reply:
Ok, validation added for `base64`.  Wanted to keep scope limited so didn't venture off into adding other content encoding validations quite yet.  What do you think?

Reply:
Fair enough. Why not raise issue(s) for the other encodings?

Reply:
Deal! https://github.com/deislabs/cnab-go/issues/132